bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -4502,21 +4502,21 @@ spec:
       name: csm-authorization-storage
     - image: quay.io/dell/container-storage-modules/csm-authorization-controller:v2.0.0
```

Comment:
Is csm-authorization not bumping a version?

Reply:
updated